### ISSUE TITLE
Mac OS upgrade to Big Sur broke SSHFS

### ISSUE THREAD

**Issue by achhabr1** [osxfuse/sshfs#68]
Seeing below error 

(python38-venv) ac043s@NJML01AC043S paloalto % sudo sshfs ac043s@10.10.57.191:/home/ ~/Documents/csvt-flexware-js 
kext load failed: -603946986
mount_macfuse: the file system is not available (255)
(python38-venv) ac043s@NJML01AC043S paloalto %

**Comment by bfleischer** [comment]
Please make sure to update to the latest version of macFUSE (currently version 4.1) and macOS 11.3.1.

**Comment by achhabr1** [comment]
I am using sshfs 2.5.0

**Comment by bfleischer** [comment]
Yes, but sshfs is based on macFUSE and you need an up-to-date version of macFUSE, that supports Big Sur, to use sshfs on Big Sur.

**Comment by designbyadrian** [comment]
I have this issue with macFUSE 4.1.2

OS X 11.5.2

**Comment by runeksvendsen** [comment]
Perhaps it's because you run the `sshfs` command as root?
